### ISSUE TITLE
Error when saving "Sendout Settings" without a selected contact

### ISSUE THREAD

**Issue by johnfmorton** [putyourlightson/craft-campaign#334]
I'm on Craft CMS 4.2.3 and installed Campaign Pro just now. 

In the initial setup process, I added email settings successfully and then moved to the "Sendout Settings" screen next. The only thing I tried to change on this screen was the "Memory Limit." I _did not_ add a contact. 

Hitting "Save" shows an error screen with the error:

```
Cannot assign string to property putyourlightson\campaign\models\SettingsModel::$defaultNotificationContactIds of type ?array
```
This happens on line 280 of putyourlightson/craft-campaign/src/controllers/SettingsController.php.

<img width="1281" alt="2022-10-13_17-08-15" src="https://user-images.githubusercontent.com/119723/195712195-a25fbccb-1351-42e5-bbff-9730683ce83b.png">

**Comment by mattstein** [comment]
Why hello, @johnfmorton.

I can reproduce this with Craft 4.2.5.1 and Campaign Lite 2.1.9, specifically from **Campaign** → **Settings** → **Sendout Settings** saving while **Default Notification Contacts** is empty. (This doesn’t happen while saving Sendouts themselves, just that default setting.)

It looks like `SettingsModel::$defaultNotificationContactIds` is getting an empty string when it wants `null` or an array.

I’ll let Qualified People respond, and for now suggest that you simply Don’t Do That™.

**Comment by johnfmorton** [comment]
Thank you. Yes, I can get around it. Just reporting and doing my civic duty.

**Comment by bencroker** [comment]
Thanks for reporting this, I'll be releasing a fix soon.

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-campaign/commit/f8db9f1e8ea6aea79ab2710157b77b549cad7de3 and released in version 2.1.10.
